irods/manager/data_object_manager.py

```diff
@@ -330,6 +330,26 @@ def open(self, path, mode, create = True, finalize_on_close = True, **options):
         return io.BufferedRandom(raw)
 
 
+    def trim(self, path, **options):
+
+        message_body = FileOpenRequest(
+            objPath=path,
+            createMode=0,
+            openFlags=0,
+            offset=0,
+            dataSize=-1,
+            numThreads=self.sess.numThreads,
+#           oprType=oprType,
```

Comment:
Is this needed?

Reply:
Don't know if an oprType is in general needed for a trim. I know `rsDataObjTrim` calls `dataObjUnlinkS`.

Theoretically, would a client ever need to pass an oprType to a TRIM operation ?